addons/medical_damage/stringtable.xml

```diff
@@ -687,5 +687,11 @@
             <Polish>Ustawia próg obrażeń jakie może otrzymać jednostka przed utratą przytomności. (0 jako ustawienie domyślne misji)</Polish>
             <Turkish>Bilinçsiz duruma geçmeden önce bir birimin alabileceği hasar miktarını ayarlar. (Görev varsayılanı için 0)</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_Damage_painUnconsciousChance_DisplayName">
+            <English>Pain Unconscious Chance</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_Damage_painUnconsciousChance_Description">
+            <English>Sets the chance of falling unconscious if pain is above the threshold.</English>
```

Comment:
For other "chance settings" we use "The probability..." and I think for consistency we should have the same here. Something like: "The probability of a person falling unconscious when their pain is above the tolerance threshold."